Assets/_BForBoss/_Core/Scripts/Debug/DebugView/FreeCamera.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class FreeCamera : DebugView
```

Comment:
Rename this to `FreeCameraDebugView` since this is not the FreeCamera itself, but the View that shows debug instrctions

Reply:
The [task](https://perigongames.atlassian.net/browse/BFB-345?atlOrigin=eyJpIjoiMDNlZTc5NGQ4NDIxNDk5N2IwN2U0MTM2NzRmOTRkODUiLCJwIjoiaiJ9) to tackle this as well as the other DebugView Class names